philips_hue_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for functions in philips_hue.py."""
+
+import pytest
+
+import main
+import philips_hue
+import philips_hue_mock
+
+
+@pytest.fixture
+def config():
+    main.app.config.from_object('config.DevConfig')
+    return main.app.config
+
+
+@pytest.fixture
+def philips_hue_client(config):
+    philips_hue_client = philips_hue.PhilipsHueClient(
+        config['BRIDGE_IP_ADDRESS'], config['USERNAME'])
+    return philips_hue_client
+
+
+def test_set_color(philips_hue_client, requests_mock):
```

Comment:
Nit: I was always given the advice that tests should attempt to tell a story. What if we format the linebreaks like:

```
# All of your setup code, including the mock.

# The system under test

# Assertions
```

So then it's clear to the reader what is the boilerplate setup versus what we are actually testing.

Reply:
Done